workspaces/libnpmpublish/package.json

```diff
@@ -44,7 +44,7 @@
     "npm-registry-fetch": "^14.0.3",
     "proc-log": "^3.0.0",
     "semver": "^7.3.7",
-    "sigstore": "^1.0.0",
```

Comment:
npm cli team needs to make this change in `latest`

Reply:
Looks like there was some work towards this in https://github.com/npm/cli/commit/fae5e0063215e97ee18a60f8136a06045d621ec2.

Anything I can do to help this along? 👀

Reply:
It already landed in latest at that point, I think if you rebase your PR against `latest` it should clear.

Reply:
I don't know if https://github.com/npm/cli/commit/fae5e0063215e97ee18a60f8136a06045d621ec2 covers the libnpmpublish workspace? package.json still shows 1.0.0 at latest: https://github.com/npm/cli/blob/0783cff9653928359a6c68c8fdf30b9fd02130c9/workspaces/libnpmpublish/package.json#L47

Reply:
Oh right thanks, got my wires crossed there